common/chat/conversation.ts

```diff
@@ -85,6 +105,14 @@ const getOrCreateOneOnOneConversation = async (
 
     const participantsConversationId = getConversationId(participants);
     const participantsConversation = await getConversation(participantsConversationId);
+    if (participantsConversation) {
+        const { readMembers } = checkChatMembersAccessRights(participantsConversation);
+        const isChatReadOnly = readMembers.length > 0;
+
+        if (isChatReadOnly) {
```

Comment:
Why do we mark readonly chats as writable?

Reply:
the idea is actually that if I have had a chat with a person before, e.g. because the match was dissolved 30 days ago or I no longer attend a course from the instructor, and therefore was set to readonly, the "old" chat will be reactivated.

Reply:
does it make sense?

Reply:
Ah I see, makes sense :)